### PR TITLE
Add Elixir.tmbundle

### PR DESCRIPTION
**Pull request by jish**
I have never created a Homebrew Cask before so I'm not sure if this is
correct or not. I used some examples in this repo and I also used the
"Adding a Cask" documentation.

Please let me know what I should change or how I should test this to
make sure it works properly.

### CODE REVIEW

File: Casks/tm-elixir.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask "tm-elixir" do
+  version :latest
+  sha256 :no_check
+
+  url "https://github.com/elixir-lang/elixir-tmbundle/archive/master.tar.gz"
+
+  name "A TextMate Bundle for the Elixir programming language."
+  homepage "https://github.com/elixir-lang/elixir-tmbundle"
+
+  tmbundle "Elixir.tmbundle"
```
Comment (claui) [inline Casks/tm-elixir.rb]: This line causes the following error:

> ==> Installing Cask tm-elixir
> Error: It seems the TextMate bundle source '/usr/local/Caskroom/tm-elixir/latest/Elixir.tmbundle' is not there.
> Error: Install incomplete.

The reason for this is that extracting `master.tar.gz` results in a directory named `elixir-tmbundle-master` while Homebrew looks for something called `Elixir.tmbundle`, which is obviously not there.

To fix this, you can just edit this line so it says:

    tmbundle "elixir-tmbundle-master", target: "Elixir.tmbundle"

That way, Homebrew will find the source directory by the odd name. Also, the `:target` option will instruct it to rename the directory to `Elixir.tmbundle` proper inside TextMate’s `Bundles` directory.

**Comment by claui** [comment]
Great work @jish, and even more so for a first Cask!

The best option for Casks is to always pin to a version, or release; this allows users to benefit from (and ultimately trust) the SHA sums. If we’re dealing with a Cask that must use `version :latest` (e. g. when the upstream project does not define releases and version numbers), then the Cask needs`sha256 :no_check`, which you did correctly. 👍

As soon as the `It seems the TextMate bundle source 'Elixir.tmbundle' is not there` error is fixed, there’s nothing stopping us from merging!

**Comment by jish** [comment]
Thanks for the help, I updated the `tmbundle` line.

**Comment by claui** [comment]
Merged.

**Pull request merged**